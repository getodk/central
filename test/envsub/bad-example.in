${NOT_DEFINED_1} ${NOT_DEFINED_2}

${NOT_DEFINED_3}
