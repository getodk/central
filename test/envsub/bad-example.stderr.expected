ERR: var not defined on line 1: ${NOT_DEFINED_1}
ERR: var not defined on line 1: ${NOT_DEFINED_2}
ERR: var not defined on line 3: ${NOT_DEFINED_3}

3 error(s) found.
